internal/metrics/metrics.go

```diff
@@ -68,5 +84,106 @@ func initMetricCollectors(reg *prometheus.Registry) {
 
 	processCollector = collectors.NewProcessCollector(collectors.ProcessCollectorOpts{})
 
-	reg.MustRegister(buildInfo, goCollector, processCollector)
+	// region API Server Metrics.
+	APIConnectionsActiveTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "api_connections_active_total",
```

Comment:
I updated the name to `api_connections_active_total` instead of `api_connections_active` to align with the [naming convention](https://prometheus.io/docs/practices/naming/#metric-names)

>...SHOULD have a suffix describing the unit, in plural form. Note that an accumulating count has **total** as a suffix